replica/table.cc

```diff
@@ -628,6 +628,13 @@ compaction_group& table::compaction_group_for_token(dht::token token) const noex
     return ret;
 }
 
+boost::integer_range<size_t> table::compaction_groups_for_token_range(dht::token_range tr) const noexcept {
+    size_t start = tr.start() ? compaction_group_for_token(tr.start()->value()).group_id() : size_t(0);
+    size_t end = tr.end() ? compaction_group_for_token(tr.end()->value()).group_id() : (_compaction_groups.size() - 1);
```

Comment:
This doesn't respect inclusive/exclusive bounds.

Reply:
yes, better not to assume inclusiveness. note only the groups in the edges may be incorrectly included, every other group would be fully contained in the supplied range. will fix, thanks.